src/fr/scanmanga/src/eu/kanade/tachiyomi/extension/fr/scanmanga/ScanManga.kt

```diff
@@ -46,23 +41,36 @@ class ScanManga : ParsedHttpSource() {
         }.build()
 
     private val json: Json by injectLazy()
+    private fun shuffle(s: String): String {
+        val chars = s.toMutableList()
+        chars.shuffle()
+        return chars.joinToString("")
+    }
```

Comment:
Is this just a refactor, or did the behavior change?

Reply:
> Is this just a refactor, or did the behavior change?

Just a refactor. (And a bonus optimization)